worlds/witness/items.py

```diff
@@ -57,9 +59,17 @@ def __init__(self):
 
             item_tab[item[0]] = ItemData(158000 + item[1], True, False)
 
+            self.ITEM_NAME_GROUPS.setdefault("Symbols", set()).add(item[0])
+
         for item in StaticWitnessLogic.ALL_DOOR_ITEMS:
             item_tab[item[0]] = ItemData(158000 + item[1], True, False)
 
+            if item[1] in range(1500, 1512):
+                self.ITEM_NAME_GROUPS.setdefault("Lasers", set()).add(item[0])
+                continue
+
+            self.ITEM_NAME_GROUPS.setdefault("Doors", set()).add(item[0])
```

Comment:
since this is an either/or with only two cases, i believe it would be easier to do
```suggestion
            else:
                self.ITEM_NAME_GROUPS.setdefault("Doors", set()).add(item[0])
```

Reply:
I may love guard clauses a bit too much.